### PR TITLE
Task #452 Day 1 and Day 2 

### PR DESCRIPTION
**Pull request by dhrutikaila**

### CODE REVIEW

File: SQL/Dhruti/Day2/SQL UPDATE.sql [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+--6				  
+UPDATE Employees SET Salary=8000 WHERE EmployeeID=105 AND Salary<5000;
+--7
+UPDATE Employees SET JobID='SH_CLERK' WHERE EmployeeID=118 AND DepartmentID=30 AND Jobid<>'SH%';
+--7
+UPDATE Employees
+SET JobId = 'SH_CLERK' 
+WHERE (EmployeeID = 118 and DepartmentID = 30 and JobId <> 'SH%')
+--8
+UPDATE Employees
```
Comment (radixdt12020) [inline SQL/Dhruti/Day2/SQL UPDATE.sql:33]: USE CASE STATEMENT

File: SQL/Dhruti/Day2/SQL UPDATE.sql [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+UPDATE Employees SET email='not work',
+CommissionPct=0.10
+--2
+UPDATE Employees SET email='not work',
+CommissionPct=0.10
+WHERE DepartmentId=80 AND CommissionPct<.20;
+--3
+UPDATE Employees SET Salary=8000 WHERE EmployeeID=2 AND Salary<5000;
+--4
+ UPDATE Employees SET EmployeeID='SH_CLERK'
```
Comment (radixdt12020) [inline SQL/Dhruti/Day2/SQL UPDATE.sql:12]: please check again with question> wRITE QUERY IN COMMENTS

**Comment by dhrutikaila** [comment]
Changes

**Comment by dhrutikaila** [comment]
Done

**Pull request closed without merging**